pdr_backend/util/subgraph_predictions.py

```diff
@@ -0,0 +1,211 @@
+from typing import List
+import json
+from enum import Enum
+
+from pdr_backend.util.subgraph import query_subgraph, info_from_725
+from pdr_backend.models.prediction import Prediction
+
+
+class FilterMode(Enum):
+    CONTRACT = 1
+    PREDICTOOR = 2
+
+
+def get_subgraph_url(network: str) -> str:
```

Comment:
This does not belong in this module, because it shouldn't be specific to subgraph predictions. It either belongs in subgraph.py or networkutil.py. I recommend the latter.

Reply:
done